src/Server/LanguageServer.cs

```diff
@@ -104,7 +104,7 @@ public static Task<ILanguageServer> From(Action<LanguageServerOptions> optionsAc
             services.AddSingleton(_serializer);
             services.AddSingleton<OmniSharp.Extensions.JsonRpc.ISerializer>(_serializer);
             services.AddSingleton(requestProcessIdentifier);
-            services.AddSingleton(_reciever);
+            services.AddSingleton<IReceiver>(_reciever);
```

Comment:
probably need it registered for both `IReciever` and `ILspReciever`.

Reply:
Didn't see that itw as a different type in this ctor. Ya you're right. Will do that.

Reply:
Done.